monai/networks/nets/segresnet.py

```diff
@@ -213,30 +192,10 @@ def __init__(
         vae_estimate_std: bool = False,
         vae_default_std: float = 0.3,
         vae_nz: int = 256,
-        spatial_dims: int = 3,
-        init_filters: int = 8,
-        in_channels: int = 1,
-        out_channels: int = 2,
-        dropout_prob: Optional[float] = None,
-        norm_name: str = "group",
-        num_groups: int = 8,
-        use_conv_final: bool = True,
-        blocks_down: tuple = (1, 2, 2, 4),
-        blocks_up: tuple = (1, 1, 1),
-        upsample_mode: Union[UpsampleMode, str] = "nontrainable",
+        **kwargs,
```

Comment:
Actually, I think maybe it would be more clear to leave the args for users to check directly.

Reply:
I would generally have arguments set out in full rather than hiding a lot of things behind args/kwargs. Users rely on the signature and docstring of a constructor to know how to create an object, it's much harder when that detail is hidden.